src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/BlockListener.java

```diff
@@ -237,10 +238,16 @@ private void dropItems(BlockBreakEvent e, List<ItemStack> drops) {
                 // Disable normal block drops
                 e.setDropItems(false);
 
+                // Fixes #4051
+                Block b = e.getBlock();
+                if (BlockStorage.check(b) == null) {
```

Comment:
Do the callers of this function have the Slimefun block (or know it's null)?
Would be nice to save doing this check and just pass through as a parameter